pyEpiabm/pyEpiabm/core/place.py

```diff
@@ -87,13 +99,35 @@ def remove_person(self, person: Person):
         if person not in self.persons:
             raise KeyError("Person not found in this place")
         else:
+            group_index = self.get_group_index(person)
+            self.person_groups[group_index].remove(person)
+            # If anyone can think of a better way to remove
+            # people from a dictionary without knowing the
+            # key lmk.
```

Comment:
The key is `group_index` isn't it? This looks sensible!

Reply:
Ah that's a left over doc string